.eslintrc.js

```diff
@@ -0,0 +1,46 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+    es6: true,
+  },
+  extends: ['plugin:react/recommended'],
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 'latest',
+    sourceType: 'module',
+  },
+  plugins: ['prettier', 'react'],
+  rules: {
+    'import/no-extraneous-dependencies': 'off',
+    'import/prefer-default-export': 'off',
+    'lines-between-class-members': ['error', 'always'],
+    'no-comma-dangle': 'off',
+    'no-multiple-empty-lines': ['error', { max: 1 }],
+    'no-unneeded-ternary': 'error',
+    'react/jsx-sort-props': 'error',
+    'react/jsx-uses-react': 'error',
+    'react/jsx-uses-vars': 'error',
+    'react/function-component-definition': 'off',
+    'react/jsx-one-expression-per-line': 'off',
+    'react/jsx-filename-extension': [
+      1,
+      {
+        extensions: ['.js', '.jsx', '.ts', '.tsx'],
+      },
+    ],
+    'react/no-unknown-property': 'off',
+    'react/react-in-jsx-scope': 'off',
+    'react/self-closing-comp': [
+      'error',
+      {
+        component: true,
+        html: false,
```

Comment:
is there a reason why we want to have different behavior for components versus html tags here?

Reply:
No, IMO, we should use the default which is true for both so I'll remove this rule.